llvm/include/llvm/Object/Wasm.h

```diff
@@ -239,7 +239,7 @@ class WasmObjectFile : public ObjectFile {
   bool isValidSectionSymbol(uint32_t Index) const;
   wasm::WasmFunction &getDefinedFunction(uint32_t Index);
   const wasm::WasmFunction &getDefinedFunction(uint32_t Index) const;
-  wasm::WasmGlobal &getDefinedGlobal(uint32_t Index);
+  const wasm::WasmGlobal &getDefinedGlobal(uint32_t Index) const;
```

Comment:
Is this an independent cleanup?

Reply:
No, because `getSymbolSize` is const, and it now calls `getDefinedGlobal`.